{"uid":"4b4757e66a1912dae1a509f688f20b0f","name":"categories","children":[{"name":"Test defects","children":[{"name":"requests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=8080): Max retries exceeded with url: /ClientTest/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000240CE92B620>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))","children":[{"name":"Test Empty Payer Name Validation","uid":"43ff6696cbce9f1c","parentUid":"4b059863b3965949d7fea2fa97bb0ab6","status":"broken","time":{"start":1742649699712,"stop":1742649699712,"duration":0},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":20,"retriesStatusChange":false,"parameters":[],"tags":["critical","ui"]}],"uid":"4b059863b3965949d7fea2fa97bb0ab6"}],"uid":"bdbf199525818fae7a8651db9eafe741"}]}